imsv-docs-astro/src/styles/tailwind.css

```diff
@@ -0,0 +1,74 @@
+@layer base {
+  :root {
+    --shiki-color-text: theme('colors.white');
+    --shiki-token-constant: theme('colors.emerald.300');
+    --shiki-token-string: theme('colors.emerald.300');
+    --shiki-token-comment: theme('colors.zinc.500');
+    --shiki-token-keyword: theme('colors.sky.300');
+    --shiki-token-parameter: theme('colors.pink.300');
+    --shiki-token-function: theme('colors.violet.300');
+    --shiki-token-string-expression: theme('colors.emerald.300');
+    --shiki-token-punctuation: theme('colors.zinc.200');
+  }
+
+  [inert] ::-webkit-scrollbar {
+    display: none;
+  }
+
+  /*
+   * Tailwind preflight is disabled by the starlight tailwind plugin but some
+   * styles are still needed. If tailwind components are behaving weird then there
+   * might be other base styles missing.
+   *
+   * If in future we replace the Starlight left nav or top header components
+   * with custom components then restoring preflight in the tailwind config may be
+   * a better option.
+   *
+   * See: https://github.com/tailwindlabs/tailwindcss/blob/f57678e29ba7a9b7b2c1c5862bc9d4fd5b03d240/src/css/preflight.css
+   */
+  img,
+  video {
+    max-width: 100%;
+    height: auto;
+  }
+  a {
+    color: inherit;
+    text-decoration: inherit;
+  }
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    font-size: inherit;
+    font-weight: inherit;
+  }
+  blockquote,
+  dl,
+  dd,
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  hr,
+  figure,
+  p,
+  pre {
+    margin: 0;
+  }
+
+}
+
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer components {
+  /* make mermaid diagrams readable on dark mode */
+  [data-theme="dark"] p img[src*=".svg"] {
+    filter: invert(0.7);
+  }
+}
```

Comment:
Do we still need this now mermaid diagrams are all replaced?

Reply:
We do, the change with mermaid was that we store diagrams in SVG and not mermaid language that is processed by plugin. This CSS snippet styles SVGs